authapi/views.py

```diff
@@ -83,7 +92,7 @@ def get_queryset(self):
         We also have the query params permission_contains and object_id, which
         allow users to filter the teams based on the permissions they
         contain.'''
-        queryset = self.queryset
+        queryset = super(TeamViewSetNoCreate, self).get_queryset()
         if self.action == 'list':
             archived = get_true_false_both(
                 self.request.query_params, 'archived', 'false')
```

Comment:
A little bikesheddy, but I'm wondering if we could create abstractions that match what the resources/view sets mean. Something like:

```
BaseTeamViewSet  # things that apply for all team view sets (e.g. get_queryset())
OrganizationTeamViewSet  # the view set for `/organizations/<id>/teams/
GlobalTeamViewSet  # the view set for `/teams/
```

Thoughts?